docs/admin/install.rst

```diff
@@ -336,9 +336,77 @@ This command will take a considerable amount of time and approximately 4GB of ba
+
+.. _Password Management Section:
+
+Password management
+~~~~~~~~~~~~~~~~~~~
+If you have installed Qubes with the default configuration, you should have a VM called ``vault``, which is a networkless VM based on the ``fedora-30`` template. You can use this VM to store SecureDrop login credentials or the passphrases for encrypted export USB drives.
+
+Note that all users of this workstation will have access to the password database. If the workstation will be used by more than one journalist, you may want to encourage the use of a smartphone password manager for managing individual login credentials.
+
+The ``fedora-30`` template includes the KeePassXC password manager. It is not listed in the application menu for ``vault`` by default; you can add it by selecting it from the list of available apps in **Q > Domain: vault > Qube Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
+
+After confirming this change, you can launch KeePassXC from the **Domain: vault** menu. On first start, it will suggest enabling automatic updates, which you should decline: ``vault`` is networkless, so the built-in update check will fail; the app will be updated through system updates instead.
+
+When creating a new password database, KeePassXC will prompt you to protect it with a password, and will refuse to accept a blank password. If you want to configure easy access to the password database, you can protect the database using a key file, via **Add additional protection > Add Key File > Generate** on the final screen. Once you have configured a key file, KeePassXC will accept leaving the master password field blank.
+
+Clipboard configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+Qubes supports copying the clipboard contents from one VM to another using `special keyboard shortcuts <https://www.qubes-os.org/doc/copy-paste/>`_. It also permits configuring the access policies for each VM's clipboard. By default, cross-VM clipboard copying is completely disabled for all VMs created as part of the installation (the ones that start with ``sd-``) for security reasons.
+
+..  important::
+
+    Before changing the clipboard configuration, we recommend reviewing the section :doc:`Managing Clipboard Access <managing_clipboard>` of this guide, which goes into further detail on the security considerations.
+
+If you did set up a password manager in the previous step, you may want to enable copying passwords from KeePassXC to the SecureDrop Client. The password manager runs in the networkless ``vault`` VM, and the SecureDrop Client runs in the ``sd-app`` VM. To permit this one-directional clipboard use, issue the following command in ``dom0``:
 
-Once the update check is complete, the client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+.. code-block:: sh
```

Comment:
There's a lot of overlap in the commands in the `Managing clipboard access` section, and it might make sense to consolidate the examples (for example move this one to the latter section and link to it). The language used here combined with the linking to information found later in the document will make it unlikely that a user will actually read the details prior to setting up clipboard sharing

Reply:
I would prefer not to decouple the concrete instructions for setting up password copying from the setup of the password manager -- those two really belong together, as most users will want both to make meaningful use of the password manager. Additionally, the risks of this single pathway for clipboard copying (one way, from ``vault`` to ``sd-app``) seem among the most manageable, especially if we land the password-wipe logic in the client along with this change.

Reply:
I'd agree with the original reviewer on this one. Folks should be made aware of the associated risks before the process to add/remove tags is presented, and since there's a dedicated page for that it would make sense to consolidate it IMO. (Also gives us a single location to update if this process changes in the future.)

Reply:
To keep a task-focused installation guide in a single section of the docs, what I've done in dd9242951e399f860eae68bf2ed42b75cf0d11ad is reduce the instructions for setting up password copying (which is one of the safer operations, and almost certainly desirable if you're using a password manager) to a minimum. If you strongly feel that it should not be in the install guide, feel free to push a commit to consolidate it, and I can review.

Reply:
Given time constraints, will think about this some more and maybe do a follow-up PR.